wordlists.js

```diff
@@ -0,0 +1,45 @@
+// browserify by default only pulls in files that are hard coded in requires
+// In order of last to first in this file, the default wordlist will be chosen
+// based on what is present. (Bundles may remove wordlists they don't need)
+var wordlists = {}
+var wordlist
+try {
+  wordlist = require('./wordlists/chinese_simplified.json')
+  wordlists.chinese_simplified = wordlist
```

Comment:
Out of interest, why not as `wordlists.chinese_simplified = require('./wordlists/chinese_simplified.json')` ?

Reply:
to get a default wordlist.

Reply:
ah, sorry, I missed this in `exports`.